webpack.config.js

```diff
@@ -84,7 +84,7 @@ var config = {
          */
         supertokensWebsite: {
             import: "supertokens-website",
-            filename: "website.js",
+            filename: "website.[hash].js",
         },
```

Comment:
i dont think we need this script actually. Since importing session should pull in supertokens-website repo as part of that bundle